2017/CG-05.md

```diff
@@ -98,6 +98,188 @@ Please register before the event [here](https://goo.gl/forms/LQVhuRuI22lK8Umw1).
 
 ## Proposal details
 
+### Threads proposal details
+
+#### i8 and i16 Value Types
+
+Tracking Issue: [i8/i16 values with limited operators #1049](https://github.com/WebAssembly/design/issues/1049)
+
+**Poll**:
+
+> Add `i8` and `i16` value types.
+
+Adding i8 and i16 value types can reduce the number of opcodes required to
+implement atomics. Because they were not considered for the initial WebAssembly
+design, adding them now introduces some potential inconsistencies.
+
+#### Encoding Scheme
+
+Tracking issue: [Encoding Scheme #12](https://github.com/WebAssembly/threads/issues/12)
+
+**Poll**:
+
+> 1. Use `0xf0` prefix byte for all threading opcodes.
+> 2. Use `0xff` prefix byte for all threading opcodes.
+
+This should be coordinated with the SIMD proposal.
```

Comment:
@lukewagner had a proposal going further, and suggesting a prefix byte followed by an LEB128. I'd add that poll before this one.